Resources/config/yui.yml

```diff
@@ -297,6 +301,23 @@ system:
                 dashboardview-ez-template:
                     type: 'template'
                     path: %ez_platformui.public_dir%/templates/dashboard.hbt
+                ez-dashboardblockbasedview:
+                    requires:
+                        - 'ez-templatebasedview'
+                        - 'dashboardblockbasedview-ez-template'
+                    path: %ez_platformui.public_dir%/js/views/ez-dashboardblockbasedview.js
+                dashboardblockbasedview-ez-template:
+                    type: 'template'
+                    path: %ez_platformui.public_dir%/templates/dashboard/dashboard.hbt
+                ez-base-dashboardblockview:
```

Comment:
as stated in a previous comment, since the view is called `eZ.DashboardBlockBaseView` the corresponding module should be called `ez-dashboardblockbaseview`